DataFormats/EcalRecHitSoA/interface/EcalUncalibratedRecHit_soa.h

```diff
@@ -0,0 +1,68 @@
+#ifndef DataFormats_EcalRecHitSoA_interface_EcalUncalibratedRecHit_soa_h
+#define DataFormats_EcalRecHitSoA_interface_EcalUncalibratedRecHit_soa_h
+
+#include <vector>
+#include <array>
+
+#include "DataFormats/EcalDigi/interface/EcalDataFrame.h"
+#include "DataFormats/EcalRecHitSoA/interface/RecoTypes.h"
+
+namespace ecal {
+
+namespace Tag {
+
+struct soa {};
+struct aos {};
```

Comment:
do I read the code correctly, that the `aos` is never used ?
if so we could clean up the code from it.

Reply:
removed in #335